### PR TITLE
Functionality to mark a product as favorite

### PR DESCRIPTION
**Pull request by netotz**
It's now possible for a user to add any product to its favorites.

### CODE REVIEW

File: Elecritic/Pages/ProductPage.razor.cs [hunk 1]
```diff
@@ -48,13 +95,9 @@ public partial class ProductPage {
...
-            bool publicationSucceeded = await ProductContext.InsertReviewAsync(review);
-            if (publicationSucceeded) {
-                PublicationMessage = "¡Reseña publicada con éxito!";
-            }
-            else {
-                PublicationMessage = "Lo sentimos, tu reseña no pudo ser publicada :(";
-            }
+            var publicationSucceeded = await ProductContext.InsertReviewAsync(review);
+            PublicationMessage = publicationSucceeded ?
+                "¡Reseña publicada con éxito!" : "Lo sentimos, tu reseña no pudo ser publicada :(";
```
Comment (Diegobr21) [inline Elecritic/Pages/ProductPage.razor.cs:100]: C# wizardry I still don´t understand fully, but Im guessing it works as the if else statement

**Review by Diegobr21 — APPROVED** [review]
Seems fine

File: Elecritic/Pages/ProductPage.razor.cs (hunk 1 above)
Comment (netotz) [reply Elecritic/Pages/ProductPage.razor.cs:100]: > Im guessing it works as the if else statement

indeed it does, it's the ternary conditional operator, see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

Comment (Diegobr21) [reply Elecritic/Pages/ProductPage.razor.cs:100]: > > Im guessing it works as the if else statement
> 
> indeed it does, it's the ternary conditional operator, see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

Nice, thank you, I liked the mnemonic device to remember it: is this condition true ? yes : no

**Pull request merged**